src/material/datepicker/calendar-range-selection-strategy.ts

```diff
@@ -21,6 +21,12 @@ export const MAT_CALENDAR_RANGE_SELECTION_STRATEGY =
 export interface MatCalendarRangeSelectionStrategy<D> {
   /** Called when the user has finished selecting a value. */
   selectionFinished(date: D | null, currentRange: DateRange<D>, event: Event): DateRange<D>;
+
+  /**
+   * Called when the user has activated a new date (e.g. by hovering over
+   * it or moving focus) and the calendar tries to display a date range.
```

Comment:
Can we list what events are forwarded to this method? (same for the method above)